java/ql/lib/ext/java.util.model.yml

```diff
@@ -360,21 +362,34 @@ extensions:
       pack: codeql/java-all
       extensible: neutralModel
     data:
-      - ["java.util", "Collections", "emptyList", "()", "manual"]
+      - ["java.util", "ArrayList", "ArrayList", "(int)", "manual"]
+      - ["java.util", "ArrayList", "size", "()", "manual"]
+      - ["java.util", "Collection", "isEmpty", "()", "manual"]
       - ["java.util", "Collection", "size", "()", "manual"]
+      - ["java.util", "Collections", "emptyList", "()", "manual"]
+      - ["java.util", "Collections", "emptyMap", "()", "manual"]
+      - ["java.util", "Collections", "emptySet", "()", "manual"]
       - ["java.util", "Iterator", "hasNext", "()", "manual"]
+      - ["java.util", "List", "clear", "()", "manual"]
```

Comment:
We probably need a comment here stating that this neutral model is introduced since `WithoutElement` has not yet been implemented for Java.
When `WithoutElement` gets implemented, we should get a summary like
`List.clear : Argument[-1].WithoutElement -> Argument[-1]`

Reply:
Is that something you need in general for Java? I can give it a go, if you think you can use it.

Reply:
I was just curious. I'm certain it's a generally useful feature, but I don't have any specific cases in mind right now.